app/models/concerns/hostext/search.rb

```diff
@@ -20,6 +20,8 @@ module Search
       scoped_search :on => :owner_type,    :complete_value => true, :only_explicit => true
       scoped_search :on => :owner_id,      :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
       scoped_search :on => :id,            :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
+      loader_values = Operatingsystem.all_loaders_map.keys
+      scoped_search :on => :pxe_loader, :complete_value => loader_values.map { |v| v.tr(' ', '_') }.zip(loader_values).to_h.symbolize_keys, :only_explicit => true
```

Comment:
Isn't this essentially the same, but more efficient?
```suggestion
      scoped_search :on => :pxe_loader, :complete_value => loader_values.map { |v| [v.tr(' ', '_').to_sym, v] }.to_h, :only_explicit => true
```

Reply:
yes, updated, thanks!